src/environments/environment.prod.ts

```diff
@@ -22,5 +22,7 @@ export const environment = {
   currencySymbol: 'EWT',
   blockExplorerUrl: 'https://explorer.energyweb.org',
   SENTRY_ENVIRONMENT: 'production',
+  INFURA_PROJECT_ID: '2DD5GcJRpMeuqMoKsgT95AQ8OI3',
```

Comment:
I will update these values when we have dedicated projects

Reply:
If this is sensitive data, then we should put it into Github variables.